build.gradle

```diff
@@ -2,7 +2,7 @@
 plugins {
     id 'com.android.application' version '7.2.0' apply false
     id 'com.android.library' version '7.2.0' apply false
-    id 'org.jetbrains.kotlin.android' version '1.6.21' apply false
+    id 'org.jetbrains.kotlin.android' version '1.7.0' apply false
```

Comment:
@hoduchaib 
với version 1.7.0 này kotlin cải tiến chính liên quan đến gì e?

Reply:
dạ cái này nó tự update ấy anh

Reply:
ở bản 1.7.0 thì kotlin có một số cải tiến:
- Cải thiên performance bằng việc tung ra bản alpha của Kotlin / JVM K2 compiler.
- Update liên quan đến biên dịch tăng dần trong gradle.
- Có thể sử dụng dấu "_" để thay thế cho type agurment có thể suy luận.
- Ổn định(Stable) các chức năng như khai báo non-nullable types, Opt-in requirements, Builder inference
- Ngoài ra thì nó còn cho phép triển khai ủy quyền(implementation by delegation) với inline value trong inline class